webpack/ForemanTasks/Components/TasksDashboard/TasksDashboardHelper.js

```diff
@@ -73,6 +73,23 @@ export const resolveQuery = query => {
   const { search } = uri.query(true);
 
   const data = { search, ...uriQuery, page: 1 };
+  const { $, tfm } = window;
+
   uri.query(URI.buildQuery(data, true));
-  window.location.href = uri.toString();
+  tfm.tools.showSpinner();
+  $.ajax({
```

Comment:
Using the `ajax` function from `jquery` considre wrong, in webpack we prefer to use the shared: https://github.com/theforeman/foreman/blob/ac1fe0de56f29d834a54a96d069231a823878ed3/webpack/assets/javascripts/react_app/API.js

* It might not work for this case because the `API` designed to receive jsons.